<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Seuran nimi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>90</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Jakeluosoite</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>70</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Postinumero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>90</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Postitoimipaikka</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompanyNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompanyPostalAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompanyZIPLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editCompanyCityLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editCompanyCancelPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="editCompanySavePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editCompanyNameLineEdit</tabstop>
  <tabstop>editCompanyPostalAddressLineEdit</tabstop>
  <tabstop>editCompanyZIPLineEdit</tabstop>
  <tabstop>editCompanyCityLineEdit</tabstop>
  <tabstop>editCompanyCancelPushButton</tabstop>
  <tabstop>editCompanySavePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
